README.md

```diff
@@ -1 +1,31 @@
-# Portfolio
\ No newline at end of file
+# Portfolio
+
+Anja Schmidt's Portfolio
+
+This is my personal portfolio website featuring all the web development projects I have realized.
+
+<img width="710" alt="Screenshot_mobile_portfolio" src="https://user-images.githubusercontent.com/75140420/138000867-c3cb9ce7-f500-46c1-8509-814e714c0e4c.png">
+
+
+So far, the website consists only of the mobile version using the following languages:
+
+HTML
+CSS
+
+👤 Anja
+
+GitHub: @Anjacodes
+Twitter: @Anja_Schmidt7
+LinkedIn: Anja Schmidt
+
+🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page. Show your support
+
+Give a ⭐️ if you like this project! Acknowledgments
+
+📝 License
+
+This project is MIT licensed.
```

Comment:
- [ ] Please make sure you follow the microverse readme [template](https://github.com/microverseinc/readme-template) and fill out the available information related to your project. Some vital information like; step to set up your project locally, link the license, acknowledgments etc is missing. Please also make sure the sub-headings are formatted appropriately as it makes your readme file more presentable.